### PR TITLE
Round image size to a multiple of 512 in the OCS case

### PR DESCRIPTION
**Pull request by maya-r**
Sometimes we test for before/after equality of size, and it seems
like OCS sometimes has a <10 byte differences in the way we
calculate space usage. Those are probably insignificant as we
don't even do sub-512 byte accesses for a VM.

**What this PR does / why we need it**:
Fixes some of the flakes in #5790.
Sometimes the import fails. This seems to be a different failure.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by brybacki** [comment]
if sig compute fails again, try to rebase onto latest to get this: https://github.com/kubevirt/kubevirt/pull/5790

**Comment by brybacki** [comment]
Can you elaborate on this:
"Those are probably insignificant as we don't even do sub-512 byte accesses for a VM."

**Comment by maya-r** [comment]
> Can you elaborate on this:
> "Those are probably insignificant as we don't even do sub-512 byte accesses for a VM."

Elaborated in the commit message. I hope this is fine.
Changed the equality check - might be clearer this way. I was lazy due to the absence of an "abs" function but this isn't too bad.

**Comment by maya-r** [comment]
This many tests suggests it's broken but none of those failures seem related

**Comment by brybacki** [comment]
Yes, looks like some problems in environment.

File: tests/storage/datavolume.go [hunk 1]
```diff
@@ -884,6 +884,19 @@ var _ = SIGDescribe("[Serial]DataVolume Integration", func() {
 	})
 
 	Context("Fedora VMI tests", func() {
+		imageSizeEqual := func(a, b int64) bool {
+			// Our OCS image size method probe is very precise and can show a few
+			// bytes of difference.
+			// A VM cannot do sub-512 byte accesses anyway, so such small size
+			// differences are practically equal.
+			if a-b < 512 {
```
Comment (brybacki) [inline tests/storage/datavolume.go:892]: I am not sure this is always correct 
0 - 1000 = -1000

maybe  -> https://golang.org/pkg/math/#Abs

**Comment by brybacki** [comment]
/lgtm cancel 
I do not like this equality check

File: tests/storage/datavolume.go (hunk 1 above)
Comment (maya-r) [reply tests/storage/datavolume.go:892]: haha, oops

**Review by enp0s3 — APPROVED** [review]
/approve

**Pull request merged**